assets/app/scripts/controllers/newfromtemplate.js

```diff
@@ -17,9 +17,9 @@ angular.module('openshiftConsole')
       $location.url(redirect);
     }
 
-    var dcContainers = $parse('template.controllerTemplate.podTemplate.desiredState.manifest.containers');
-    var stiBuilderImage = $parse('parameters.strategy.stiStrategy.image');
-    var outputImage = $parse('parameters.output.to.name || parameters.output.DockerImageReference');
+    var dcContainers = $parse('spec.template.spec.containers');
+    var stiBuilderImage = $parse('spec.strategy.sourceStrategy.from.name');
+    var outputImage = $parse('spec.output.to.name || spec.output.DockerImageReference');
```

Comment:
I think this is for information purposes only... update to use the API version we're expecting, and if we get empty values here (possibly because the template objects are using a different API version), make sure we tolerate them